src/main/java/com/example/services/implementations/PasswordService.java

```diff
@@ -24,4 +25,14 @@ public String generateSecurePassword(String passwordToHash, String salt) {
         }
         return generatedPassword;
     }
+
+    private String generateRandomString() {
```

Comment:
maybe better to use special function from Java?

Reply:
@Vlada0612 Done.